zerver/worker/queue_processors.py

```diff
@@ -1153,6 +1154,13 @@ def failure_processor(event: Dict[str, Any]) -> None:
             )
             user_profile = get_user_profile_by_id(event["user_profile_id"])
             reactivate_user_if_soft_deactivated(user_profile)
+        elif event["type"] == "scrub_deactivated_realms":
+            realms_to_scrub = Realm.objects.filter(
+                deactivated=True, scheduled_deletion_date__lte=timezone_now()
+            )
+            for realm in realms_to_scrub:
+                logger.info("Scrubbed realm %s", realm.id)
+                do_scrub_realm(realm, acting_user=None)
```

Comment:
I am not sure about the right value of `acting_user` here.

Reply:
`None` isn't wrong.  Alternately, we could look up who deactivated the realm last, and use that value.